src/main/java/org/ozwillo/dcexporter/service/CkanService.java

```diff
@@ -38,10 +38,10 @@
         if(!opt.isPresent()) return Either.left("dataset.notif.error.fetch_datasets");
 
         List<String> datasets = opt.get();
-        
+
```

Comment:
for the next ones, avoid the noise in the commits (changes only in space, indentation, ...)